rosa_release_notes/rosa-release-notes.adoc

```diff
@@ -0,0 +1,104 @@
+:_content-type: REFERENCE
+[id="rosa-release-notes"]
+= {product-title} release notes
+include::_attributes/attributes-openshift-dedicated.adoc[]
+:context: rosa-release-notes
+
+toc::[]
+
+{product-title} ROSA is a fully-managed, turnkey application platform that allows you to focus on delivering value to your customers by building and deploying applications. Red Hat and AWS Site reliability engineering (SRE) experts manage the underlying platform so you do not have to worry about the complexity of infrastructure management. ROSA provides seamless integration with a wide range of AWS compute, database, analytics, machine learning, networking, mobile, and other services to further accelerate the building and delivering of differentiating experiences to your customers.
```

Comment:
Should it be "AWS site reliability", without the capital S?

Reply:
Good catch, made the change.